app/assets/stylesheets/mobile/topic-list.scss

```diff
@@ -36,6 +36,11 @@
         flex: 1 1 auto;
       }
     }
+
+    .navigation-controls .bulk-select,
+    .plugin-outlet-component {
+      display: none;
+    }
```

Comment:
The bulk actions button is only broken using the new experimental site setting. See internal topic `t/134051`. 

Also, hiding the `plugin-outlet-component` might have side effects (like removing a button added by a custom plugin/theme in a custom site).

Reply:
If this is the case charlie, lets leave the bulk select button alone for now. I think we can do a design round to improve this on mobile.

Reply:
> Also, hiding the `plugin-outlet-component` might have side effects (like removing a button added by a custom plugin/theme in a custom site).

Hiding the plugin outlet was accidental, good catch.